User object omitted from integration creates

Jupith opened the pull request.

splatterxl commented on docs/topics/Gateway_Events.md:
```diff
@@ -780,7 +780,7 @@ Sent when a user has unsubscribed from a guild scheduled event.
 #### Integration Create
 
-Sent when an integration is created. The inner payload is an [integration](#DOCS_RESOURCES_GUILD/integration-object) object with an additional `guild_id` key:
+Sent when an integration is created. The inner payload is an [integration](#DOCS_RESOURCES_GUILD/integration-object) object with an additional `guild_id` key, however the `user` object is omitted from the event
```
```suggestion
Sent when an integration is created. The inner payload is an [integration](#DOCS_RESOURCES_GUILD/integration-object) object with `user` omitted and an additional `guild_id` key:
```

advaith1 committed:
Document role subscriptions (#5724)

* Document role subscriptions

* remove MONETIZATION_ENABLED guild feature

devsnek committed:
document audit log after parameter (#5821)

AlmostSuspense committed:
Improve message intent docs (#5752)

* improve msg intent docs

* fix link syntax

Jupith committed:
Add max webhooks per guild error

Mateo-tem committed:
add error 50067 (#5626)

advaith1 committed:
Document error code 40062 and add code to rate limit structure (#5574)

* Document error code 40062

* Add code to rate limit structure

drakon64 committed:
The 'value' field in 'Application Command Interaction Data Option' can return 'boolean' (#5820)

mbialecka committed:
Document nullability of forum tag properties (#5816)

AlmostSuspense committed:
improve and remove outdated warning for the a.c scope (#5812)

ippess committed:
Update Channel.md (#5799)

Jupith committed:
Add missing gateway events (#5770)

SQKo committed:
Document error code `50091` (#5746)

Error returned when trying to create stage instance with a `guild_scheduled_event_id` that has `entity_type` not for `STAGE_INSTANCE`

shaydewael committed:
fix info boxes

mbialecka reviewed (changes requested):
Please update 'Integration Update' for consistency as well.

advaith1 committed:
Document that the global rate limit falls back to IP (#5806)

johnnyataisg committed:
Add missing message types (#5815)

* Add missing message types

* Update docs/resources/Channel.md

Co-authored-by: Aura Román <kyradiscord@gmail.com>

Co-authored-by: Aura Román <kyradiscord@gmail.com>

shaydewael committed:
revert stage message types (#5826)

Kowlin committed:
Update role tags (#5771)

Co-authored-by: Magdalena Białecka <2598290+mbialecka@users.noreply.github.com>

Jupith committed:
enabled and user are present on bots (#5701)

* Small Change(s)

* Add warn

Co-authored-by: Magdalena Białecka <2598290+mbialecka@users.noreply.github.com>

advaith1 committed:
Add TikTok connection (#5602)

IllagerCaptain committed:
Update Reference.md (#5681)

shaydewael committed:
Add thread member pagination and details changes (#5834)

* Add thread member pagination and details changes

* broken links

* add note to table about member field

* formatting

* oops

jkcailteux committed:
GIF Sticker image format support (#5833)

* gif sticker support

* Update docs/Reference.md

Co-authored-by: Danny <1695103+Rapptz@users.noreply.github.com>

* update cdn url

Co-authored-by: Danny <1695103+Rapptz@users.noreply.github.com>

Zoddo committed:
Clarify that USER_UPDATE is dispatched only for the current user (#5761)

This wasn't very clear, and one could guess this can be sent when an update is made to any user.

advaith1 committed:
Document role subscription system channel flags and message field (#5828)

* Document role subscription system channel flags and message field

* cumulative

ssMMiles committed:
Clarify Gateway Ratelimiting (#5503)

Clarify that the ratelimit on sending gateway events is applied on a per shard basis. This is currently not mentioned anywhere.

> Apps can send 120 gateway events every 60 seconds

advaith1 committed:
Update GameSDK docs due to feature deprecation (#5618)

* Update GameSDK docs due to feature deprecation

* Apply suggestions from code review

Co-authored-by: shay <swdewael@gmail.com>

* fix formatting mistake

* thank you for spamming my email github

* Update docs/Change_Log.md

Co-authored-by: shay <swdewael@gmail.com>

dayo05 committed:
Add Discord Nitro Basic to Game SDK API (#5845)

MinnDevelopment committed:
Rate limit is per connection not shard (#5844)

* Rate limit is per session not shard

* Per connection

SQKo committed:
Clarify `CONNECT` permission for getting text in voice messages (#5111)

* Explicitly note about Text in Voice permission

Make it clear that these endpoints also requires the `CONNECT` permission to see the text in the voice channels.

* Update Channel.md

update wording

* Apply suggestions from code review

Co-authored-by: advaith <advaithj1@gmail.com>

* Update docs/resources/Channel.md

* Update docs/resources/Channel.md

Co-authored-by: advaith <advaithj1@gmail.com>
Co-authored-by: shay <swdewael@gmail.com>

typpo committed:
Add license files (#5863)

MinnDevelopment committed:
Specify lower bound for metadata fields (#5854)

devsnek committed:
Add GUILD_AUDIT_LOG_ENTRY_CREATE event (#5849)

hammerdr committed:
[Guild Member] Adds documentation for GuildMember.flags (#5874)

Co-authored-by: shay <swdewael@gmail.com>

shaydewael committed:
format Guild.md

jixwanwang committed:
more info about guild member flags (#5881)

* more info about guild member flags

* update more stuff

ajpalkovic committed:
Remove outdated thread default auto archive time

Fixes https://github.com/discord/discord-api-docs/issues/5873

tpcstld committed:
Document Silent Messages. (#5910)

Hey folks!

This is actually my 2022 hackweek project which I got to finish to
completion. :)

During a message send request, if you include the new
`SUPPRESS_NOTIFICATIONS` flag it will not broadcast any push/desktop
notifications but will still increment the relevant mention counters.

The intention is that you can get someone's attention but not feel like
you could be distracting them. Like when you DM someone at 5am. I'm sure some
bots can leverage this as well to avoid noise or something.

Also this should work for the webhook send request as well.

[Add a picture of the UI here]

If you're looking to leverage this as a non-bot, you can write `@silent`
as the _very first_ thing in a message. Make sure your client is
up-to-date btw. Autocomplete a-la `@everyone` is not planned. Eventually we may
put this in an "actual UI", for now this is where it lives. :)

Also sorry to all the users on Discord named silent who may have some
textual conflict now. Forgive me!

tpcstld committed:
Add suppress notifications in create message endpoints.

shaydewael committed:
Add Upcoming Locked Thread Change (#5919)

* add upcoming locked change

* update permission

* one more permission change

* another banner

* one tiny clarification

hemu committed:
[Auto Moderation] Increase keyword limits (#5909)

Co-authored-by: shay <swdewael@gmail.com>

Commit:
Document funny error code 30061 (#5923)

* Update Opcodes_and_Status_Codes.md

* Update Opcodes_and_Status_Codes.md

MinnDevelopment committed:
Add note about audit log entry order in pagination (#5843)

Co-authored-by: Suspense <64612795+AlmostSuspense@users.noreply.github.com>
Co-authored-by: snek <me@gus.host>

panley01 committed:
Document `role_connections.write` Implicit grant incompatibility (#5885)

Following fragement is provided:
`#error=invalid_scope&error_description=The+requested+scope+is+invalid%2C+unknown%2C+or+malformed.`
As JSON:
```json
{
   "error": "invalid_scope",
   "error_description": "The requested scope is invalid, unknown, or malformed."
}
```

Jupith committed:
Add guild id and ended params to app entitlements (#5853)

Jupith committed:
Max overrides when syncing (#5930)

panley01 committed:
Document error code 30011 & managed field for channels (#5867)

Commit:
add Instagram connection (#5876)

panley01 committed:
Document CDN endpoint for server subs assets (#5856)

* Document CDN endpoint for server subs assets

Server subs assets use the path: `:app_id/store/:asset_id`, while the actual asset IDs cannot be managed or fetched by bots, this should be documented for completeness in case an app ever wishes to refer to these assets.

* Updated wording & positioning

Endpoint is used for store page assets & repurposed for server subs. For consideration of legacy store pages & other implementations (like sticker pack banners), this naming & position would be more accurate.

Idris1401 committed:
redirect Discord4J to their repo (#5918)

VelvetToroyashi committed:
Clarify sticker constraints (#5852)

advaith1 committed:
Update channel type lists for stage channel updates (#5928)

* Update channel type lists for stage channel updates

* Update docs/topics/Permissions.md

Co-authored-by: Valle <bluetiger.bahnfan@gmail.com>

---------

Co-authored-by: Valle <bluetiger.bahnfan@gmail.com>

MinnDevelopment committed:
Revert "Revert "Add missing message types (#5815)"" (#5927)

* Revert "Revert "Add missing message types (#5815)""

This reverts commit 2fa10426400e420f6eb3557b5de4bd49ed55ce0b.

* Remove type 30

Commit:
Fix Cloudflare capitalization. (#5944)

CloudFlare => Cloudflare

Lulalaby committed:
docs: Add max count of embed object array (#5932)

* docs: Add max count of embed object array

This is already noted like that for webhook execute but not for message create.
The only place this information can be found right now is https://discord.com/developers/docs/change-log#support-for-multiple-embeds-in-message-routes

* docs: Rework wording

* docs: Reword once again and add info to thread

* docs: Missing space and incorrect information

* docs: Update message patch

Qjuh committed:
Document properties per channel type for create (#5941)

hemu committed:
[automod] Add custom_message metadata field (#5955)

Add new Auto Moderation custom_message action metadata field.

advaith1 committed:
fix gateway transport compression link (#5977)

JBYoshi committed:
Fix message content intent workarounds link. (#5954)

The linked article seems to have been moved to the developer support site, but this page wasn't updated to use that link.

GoldenAngel2 committed:
Update the max webhooks amount from 10 -> 15 (#5966)

codeofandrin committed:
Change keyword maximum chars in cross-reference (#5956)

shaydewael committed:
update link checker to support mdx (#6005)

shaydewael committed:
Update Getting Started Guide (#6008)

* test collapse

* Update getting started guide

* small edits

advaith1 committed:
Document automod badge app flag (#6021)

splatterxl committed:
document error code 50163 (#5948)

advaith1 committed:
Update permissions (#6017)

* Update permissions

* Update docs/topics/Permissions.md

---------

Co-authored-by: shay <swdewael@gmail.com>

Jupith committed:
Add max stage video users (#5273)

* Add max stage video users

* Update Guild.md

* lol

---------

Co-authored-by: shay <swdewael@gmail.com>

AlmostSuspense committed:
Add "discord" to disallowed names for webhooks (#5999)

* formatting and edit forbidden names

* Update docs/resources/Webhook.md

---------

Co-authored-by: shay <swdewael@gmail.com>

FuzzyGiraffee committed:
Update Gateway_Events.md (#6033)

* Update Gateway_Events.md

* Update docs/topics/Gateway_Events.md

---------

Co-authored-by: shay <swdewael@gmail.com>

hammerdr committed:
[onboarding] Adds docs for Onboarding API (#5915)

Co-authored-by: Shay <shay.dewael@discordapp.com>
Co-authored-by: Florian Spieß <business@minnced.club>

devsnek committed:
document interaction channel field (#6051)

Doc94 committed:
Fix json example for onboarding (#6063)

remove the "," added in all final of objects.

shaydewael committed:
small update (#6080)

hampuskraft committed:
Update file upload size limit (#6078)

shaydewael committed:
Bot User update (#6079)

* update references

* Change log

tpcstld committed:
Document voice messages. (#6082)

advaith1 committed:
Document join raid and mention raid protection (#5778)

Document join raid and mention raid protection

hemu committed:
Update changelog for join raid and mention raid changes (#6133)

advaith1 committed:
Document Use External Sounds permission (#6039)

Co-authored-by: shay <swdewael@gmail.com>

Jupith committed:
Owner cannot be pending (#6097)

hackermondev committed:
document error code 50178 (#6091)

hampuskraft committed:
Update invite gateway event docs (#6131)

* Update invite gateway event docs

* Update docs/topics/Gateway_Events.md

Co-authored-by: Andrin S. <65789180+Puncher1@users.noreply.github.com>

---------

Co-authored-by: Andrin S. <65789180+Puncher1@users.noreply.github.com>

codeofandrin committed:
Add Voice and Forum to `rate_limit_per_user` (#5980)

shaydewael committed:
Update Intro page for docs (#6213)

* update intro page content

* update intro smul

* add line

* small edit because reuse

* small edits

* Update docs/Intro.mdx

Co-authored-by: Jiralite <33201955+Jiralite@users.noreply.github.com>

---------

Co-authored-by: Jiralite <33201955+Jiralite@users.noreply.github.com>

advaith1 committed:
Fix formatting of recent changelog fields (#6162)

mr-tech committed:
clarify GET archived thread `before` query param (#6163)

MineBartekSA committed:
Update channel create message request size limit (#6092)

* fix: channel create message request size limit

* fix: update max request size in start thread in forum channel

shaydewael committed:
Username Docs Updates (#6218)

* start username docs

* default avatar

* Update docs/Reference.md

Co-authored-by: Florian Spieß <business@minn.dev>

* Update docs/Change_Log.md

Co-authored-by: Néstor Pérez <25409753+prryplatypus@users.noreply.github.com>

* fix docs

* reorder

* clarify migrated/legacy

* small changelog clarification

* Update docs/Change_Log.md

Co-authored-by: Florian Spieß <business@minn.dev>

* Update docs/Reference.md

Co-authored-by: Florian Spieß <business@minn.dev>

* oops

* great

---------

Co-authored-by: Florian Spieß <business@minn.dev>
Co-authored-by: Néstor Pérez <25409753+prryplatypus@users.noreply.github.com>

spifory committed:
change "lurkable" to "Discoverable" on get guild preview route (#6195)

* change "lurkable" to "Discoverable" on get guild preview route

* refactor: move the second sentence to another line

Makes more sense as opposed to slicing the sentence in half

* fix: remove the `be`

This explains why I failed English

Co-authored-by: Lala Sabathil <aiko@aitsys.dev>

* refactor: apply suggestions from code review

Co-authored-by: shay <swdewael@gmail.com>

---------

Co-authored-by: Lala Sabathil <aiko@aitsys.dev>
Co-authored-by: shay <swdewael@gmail.com>

pointydev committed:
Change URL for original snowflake documentation to twitter-archive (#6216)

While the original URL using the "twitter" organisation does redirect to the intended destination (at "twitter-archive"), the "twitter" org is exists for other purposes currently and this would make the link not rely on GitHub's redirection.

devsnek committed:
add select_voice_channel navigate param (#6227)

shaydewael committed:
Add product page best practices (#6245)

* add product page best practices

* replace image

* rename

* name

Lulalaby committed:
update emoji gateway event information (#6135)

`MESSAGE_REACTION_REMOVE` and `MESSAGE_REACTION_REMOVE_EMOJI` has the field `emoji`.`animated` now.

colinloretz committed:
Add warning about Gateway connection from Cloudflare Workers (#6246)

n0nb1narydev committed:
Update Getting_Started.mdx (#6259)

Removed invalid comment from the code

advaith1 committed:
Document role flags (#6269)

panley01 committed:
Document attachment flags (#6272)

* Document attachment flags

Documents the new attachment flags field and the flag `1 << 2` for the remix feature. I am not aware of any other flags on attachments.

* Update docs/resources/Channel.md

Co-authored-by: Lala Sabathil <aiko@aitsys.dev>

* Update docs/resources/Channel.md

Co-authored-by: advaith <advaithj1@gmail.com>

---------

Co-authored-by: Lala Sabathil <aiko@aitsys.dev>
Co-authored-by: advaith <advaithj1@gmail.com>

hammerdr committed:
Adds docs for Onboarding updates and new mode field (#6101)

* Adds docs for Onboarding updates and new mode field

* Add constraints and info

* Update docs/resources/Guild.md

Co-authored-by: shay <swdewael@gmail.com>

* add error codes for onboarding

---------

Co-authored-by: shay <swdewael@gmail.com>

MahdiBM committed:
add `message_author_id` to 'Message Reaction Add' Gateway event (#6102)

* add `message_author_id` to Message Reaction Add

* better description

* use 'who' instead of 'that'

* use 'which' instead of 'that'

hammerdr committed:
Removes stale documentation about adding roles to members bypassing verification (#6224)

LordOfPolls committed:
docs: add missing audit log events (#6071)

* docs: add missing audit log events

* docs: remove role prompt

Co-authored-by: shay <swdewael@gmail.com>

* docs: remvoe server home

Co-authored-by: shay <swdewael@gmail.com>

---------

Co-authored-by: shay <swdewael@gmail.com>

codeofandrin committed:
Add missing field `default_forum_layout` to Create Guild Channel (#5994)

Commit:
fix(typo): `Github` -> `GitHub` (#6273)

Jupith committed:
Adds approx guild count and get self application endpoint (#5971)

* Add endpoint

* Add new fields

* Update docs/resources/Application.md

Co-authored-by: IllagerCaptain <76600137+IllagerCaptain@users.noreply.github.com>

* Update docs/resources/Application.md

Co-authored-by: shay <swdewael@gmail.com>

* Update docs/resources/Application.md

Co-authored-by: shay <swdewael@gmail.com>

* Update docs/resources/Application.md

Co-authored-by: shay <swdewael@gmail.com>

* Update docs/resources/Application.md

Co-authored-by: shay <swdewael@gmail.com>

---------

Co-authored-by: IllagerCaptain <76600137+IllagerCaptain@users.noreply.github.com>
Co-authored-by: shay <swdewael@gmail.com>

kkrypt0nn committed:
Ordered table and wording consistency (#6100)

* ordered table and wording consistency

* Updated the return type of Modify Guild Widget

shaydewael committed:
make message_author_id optional (#6296)

advaith1 committed:
Document avatar decorations (#5723)

Co-authored-by: shay <swdewael@gmail.com>

ManHatos committed:
Minor Wording Fix (#6280)

* Minor Wording Fix

Bot users are now automatically attached to newly created applications. This part of the docs is outdated.

* Update docs/interactions/Receiving_and_Responding.md

Co-authored-by: shay <swdewael@gmail.com>

* Update docs/interactions/Receiving_and_Responding.md

Co-authored-by: shay <swdewael@gmail.com>

* Update docs/interactions/Receiving_and_Responding.md

---------

Co-authored-by: shay <swdewael@gmail.com>

valdotle committed:
Add changelog entry for application connection metadata and linked roles (#5851)

* Update Change_Log.md

* Update Change_Log.md

* Update Change_Log.md

* Update Change_Log.md

* Update docs/Change_Log.md

Co-authored-by: Suspense <64612795+AlmostSuspense@users.noreply.github.com>

* Update docs/Change_Log.md

* Update docs/Change_Log.md

* Update docs/Change_Log.md

---------

Co-authored-by: Suspense <64612795+AlmostSuspense@users.noreply.github.com>
Co-authored-by: shay <swdewael@gmail.com>

sinde530 committed:
Fix typo Store.md (#6235)

JohnyTheCarrot committed:
Document with_counts param for GET /users/@me/guilds (#5628)

* Document with_counts param for GET /users/@me/guilds

* Update docs/resources/Guild.md

---------

Co-authored-by: shay <swdewael@gmail.com>

colinloretz committed:
Mark optional params for PATCH channel positions (#6147) (#6301)

insacc committed:
Introduce the `GUILD_MEDIA` channel type (#6232)

* Add docs about GUILD_MEDIA channel type

* Add more information about media channels

* Create a changelog for the `GUILD_MEDIA` channel type

* Update docs/resources/Channel.md

Co-authored-by: Danny <1695103+Rapptz@users.noreply.github.com>

* Update the change log

Co-authored-by: shay <swdewael@gmail.com>

* Update the wording on the change log

Co-authored-by: shay <swdewael@gmail.com>

* Update more wording on the change log

Co-authored-by: shay <swdewael@gmail.com>

* Move info to the intro

Co-authored-by: shay <swdewael@gmail.com>

* Remove the moved info

Co-authored-by: shay <swdewael@gmail.com>

* Replace slash with `and`

Co-authored-by: shay <swdewael@gmail.com>

* Update the beta state

Co-authored-by: shay <swdewael@gmail.com>

* Fix more wording

Co-authored-by: shay <swdewael@gmail.com>

* Address more pr comments

* Address more pr comments

* Fix broken links

* Fix typo

Co-authored-by: Valle <bluetiger.bahnfan@gmail.com>

* Address more pr comments

---------

Co-authored-by: shay <swdewael@gmail.com>
Co-authored-by: Danny <1695103+Rapptz@users.noreply.github.com>
Co-authored-by: Valle <bluetiger.bahnfan@gmail.com>

insacc committed:
Update the change-log date for the media channel type (#6333)

colinloretz committed:
Fix formatting of GUILD_MEDIA changelog entry (#6334)

shaydewael committed:
add openapi changelog (#6336)

advaith1 committed:
Bots can now set activity state (#6345)

* Bots can now set activity state

* fix custom status format

* wording changes

Co-authored-by: shay <swdewael@gmail.com>

---------

Co-authored-by: shay <swdewael@gmail.com>

emmanuel-ferdman committed:
fix: typo in Gateway.md (#6335)

Signed-off-by: Emmanuel Ferdman <emmanuelferdman@gmail.com>

shaydewael committed:
embed debugger changelog (#6352)

typpo committed:
Add community resources guideline to CONTRIBUTING.md

Previously this info was buried in a github discussion

shaydewael committed:
Update teams docs (#6372)

* Update Teams docs

* small edit

* move into table

* update assets

* update other table

* add changelog

* small clarif

* table formatting

appellation committed:
Document integration_type for audit log entries (#6367)

colinloretz committed:
Clarify return values for Text Input components in an interaction (#6344)

* Clarify return values for Text Input components in an interaction

* Fix link to text-input-object-text-input-styles

* Update docs/interactions/Message_Components.md

Co-authored-by: shay <swdewael@gmail.com>

* Clarify returned attributes when using text input

Co-authored-by: shay <swdewael@gmail.com>

---------

Co-authored-by: shay <swdewael@gmail.com>

colinloretz committed:
Callout for implicit permissions for channels and guilds (#6370)

* Channel permissions callout/link to Implicit Permissions

* Guild permissions callout/link for Implicit Permissions

* fix table

* Update docs/resources/Channel.md

Co-authored-by: shay <swdewael@gmail.com>

* added missing word + table spacing

---------

Co-authored-by: Lala Sabathil <aiko@aitsys.dev>
Co-authored-by: shay <swdewael@gmail.com>

advaith1 committed:
Standard stickers are now free (#6265)

* Standard stickers are now free

* Update docs/resources/Sticker.md

Co-authored-by: shay <swdewael@gmail.com>

---------

Co-authored-by: shay <swdewael@gmail.com>

Mysterious-Dev committed:
Add Guild Navigation Message Format (#6105)

* Add Id Message Format

* Add missing types

Co-authored-by: Shadow <william.shadow@outlook.com>

* Format

* Format

* Apply suggestions

* Fix bug

* Apply suggestions

---------

Co-authored-by: Shadow <william.shadow@outlook.com>

Mateo-tem committed:
document super reactions (#6056)

* document super reactions

* delete me_burst field

Co-authored-by: Lala Sabathil <aiko@aitsys.dev>

* add error code 90002

* apply suggestions

Co-authored-by: shay <swdewael@gmail.com>

---------

Co-authored-by: Lala Sabathil <aiko@aitsys.dev>
Co-authored-by: shay <swdewael@gmail.com>

dependabot[bot] committed:
Bump word-wrap from 1.2.3 to 1.2.4 (#6309)

Bumps [word-wrap](https://github.com/jonschlinkert/word-wrap) from 1.2.3 to 1.2.4.
- [Release notes](https://github.com/jonschlinkert/word-wrap/releases)
- [Commits](https://github.com/jonschlinkert/word-wrap/compare/1.2.3...1.2.4)

---
updated-dependencies:
- dependency-name: word-wrap
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

cabbage63 committed:
Fix typo in example code of Getting Started (#6353)

Lulalaby committed:
[cleanup] markdown table formatting (#6374)

* fix table again

* Update Message_Components.md

colinloretz committed:
[housekeeping] GH Action markdown table formatting (#6376)

* Add markdown table format check to GH Actions

* Check in all docs folders for .md and .mdx

* Support globstar in bash command

* chore(actions): install markdown-table-formatter via package.json (#2)

adds the possibility to run it locally (check & fix)

* Move docs format check into its own workflow

---------

Co-authored-by: Lala Sabathil <aiko@aitsys.dev>

devsnek committed:
clarify team owner role (#6384)

* Update Teams.md

* Update Teams.md

* Update docs/topics/Teams.md

Co-authored-by: shay <swdewael@gmail.com>

---------

Co-authored-by: shay <swdewael@gmail.com>

shaydewael committed:
update issue templates (#6391)

typpo committed:
Update Community Resources

Per these guidelines https://github.com/discord/discord-api-docs/discussions/4456

dependabot[bot] committed:
Bump actions/checkout from 3 to 4 (#6402)

Bumps [actions/checkout](https://github.com/actions/checkout) from 3 to 4.
- [Release notes](https://github.com/actions/checkout/releases)
- [Changelog](https://github.com/actions/checkout/blob/main/CHANGELOG.md)
- [Commits](https://github.com/actions/checkout/compare/v3...v4)

---
updated-dependencies:
- dependency-name: actions/checkout
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

JustinBeckwith committed:
chore: include npm in dependabot config (#6415)

colinloretz committed:
Added localized Getting Started guides (#6426)

JustinBeckwith committed:
chore: bump eslint, prettier, typescript deps (#6427)

shaydewael committed:
update text for activities.write (#6440)

* update text for activities.write

* ok bye

IBXCODECAT committed:
Twitter Rebrand Update (#6394)

* Twitter > X user resource

* Fix Markdown Issues?

* Update docs/resources/User.md

Co-authored-by: Danial Raza <danialrazafb@gmail.com>

---------

Co-authored-by: shay <swdewael@gmail.com>
Co-authored-by: Danial Raza <danialrazafb@gmail.com>

dependabot[bot] committed:
Bump @types/node from 16.10.3 to 20.6.2 (#6438)

Bumps [@types/node](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/node) from 16.10.3 to 20.6.2.
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/node)

---
updated-dependencies:
- dependency-name: "@types/node"
  dependency-type: direct:development
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

ypisetsky committed:
[create_channel] Document default_thread_rate_limit_per_user (#6414)

This also works on guild channel create. Document it there too (description copied from Channel resource)

JustinBeckwith committed:
build: move to ESM and update chalk (#6441)

shaydewael committed:
Default select menu values (#6442)

* add default select docs

* small edits

* classic

* add resolved

* Update docs/interactions/Message_Components.md

Co-authored-by: Valle <81811276+Valle-infinitare@users.noreply.github.com>

* add date

* great i love this so much i love tables i love formatting

---------

Co-authored-by: Valle <81811276+Valle-infinitare@users.noreply.github.com>

colinloretz committed:
Document Premium App Subscriptions (#6451)

* WIP premium apps

* Rework Overview (#4)

* overview page

* start app subscriptions edits

* move to monetization

* jk

* Implementing App Subscriptions

* Added query params to entitlements

* Update to Monetization Overview

* Update entitlements in interactions

* Round of small edits (#5)

* round of edits

* table/link

* Use Nelly sample app. Remove en-us from help center

* Added entitlements to Interaction object

* Update to Test Entitlement

* Testing your implementation

* Added emoji/icon docs

* Added Premium Apps to change log

* fix tables

* Update docs/monetization/App_Subscriptions.md

Co-authored-by: Jiralite <33201955+Jiralite@users.noreply.github.com>

---------

Co-authored-by: shay <swdewael@gmail.com>
Co-authored-by: Jiralite <33201955+Jiralite@users.noreply.github.com>

colinloretz committed:
Added Premium Apps Policy (#6453)

Lulalaby committed:
add premium apps bug report form and fix links for others (#6454)

seailz committed:
added a note about typing indicators expiring (#6263)

* added a note about typing indicators expiring

* Update Channel.md

Co-authored-by: shay <swdewael@gmail.com>

---------

Co-authored-by: shay <swdewael@gmail.com>

shaydewael committed:
Remove Game and Server Management docs (#6458)

* remove game and server management docs

* remove dead link

Lulalaby committed:
Rework OAuth2 documentation (#6356)

- Clarified that `Get Current User Guilds` does not need an access token if it's called for a bot
- Updated the `Example Authorization Information` to comply with the rest of the docs (pomelo),
- Documented access token revocation according to the [RFC 7009 Section 2.1](https://datatracker.ietf.org/doc/html/rfc7009#section-2.1), [RFC 6749 Section 2.3](https://datatracker.ietf.org/doc/html/rfc6749#section-2.3) and discords actual implementation with an example
- Changed examples to use `auth=(CLIENT_ID, CLIENT_SECRET)`
- Removed unused var

Co-authored-by: shay <swdewael@gmail.com>
Co-authored-by: splatter <67709748+splatterxl@users.noreply.github.com>
Co-authored-by: Zoddo <github-public-contact@zoddo.fr>
Co-authored-by: Justin Beckwith <justin.beckwith@gmail.com>

FedorLap2006 committed:
Add resolved to message component data (#5590)

* Add resolved to message component data

Add missing resolved field to message component data table.
Due to recent addition of auto-populated selects,
the components might receive resolved data structures.
This change was mentioned in message component page,
however was not reflected in the table for message component data itself.

* Update docs/interactions/Receiving_and_Responding.md

Co-authored-by: Fedor Lapshin <fe.lap.prog@gmail.com>

---------

Co-authored-by: shay <swdewael@gmail.com>

MineBartekSA committed:
Update emoji and sticker create max size units (#6093)

* fix: emoji create file size unit

* fix: sticker create file types and size unit

* fix: revert create stricker file type chnages

* fix: grammatical error

---------

Co-authored-by: shay <swdewael@gmail.com>

Darker-Ink committed:
Adding version and encoding info for resuming (#5725)

* Adding version and encoding info for resuming

Solves #5702

* Update Gateway.md

* Update docs/topics/Gateway.md

* Update docs/topics/Gateway.md

---------

Co-authored-by: shay <swdewael@gmail.com>

Victorsitou committed:
feat(Stage Instace): add `guild_scheduled_event_id` field to create stage instace endpoint (#5687)

Jupith committed:
Removing lurking info from public stages (#5768)

shaydewael committed:
Update Application Endpoints (#6297)

* update application endpoints

* make bot optional

* uh

* add back approximate_guild_count

* aaa

* Update docs/resources/Application.md

Co-authored-by: Lala Sabathil <aiko@aitsys.dev>

* Update docs/resources/Application.md

Co-authored-by: Lala Sabathil <aiko@aitsys.dev>

* TABLES

* tag characters

---------

Co-authored-by: Lala Sabathil <aiko@aitsys.dev>

Lulalaby committed:
Tweak documentation for premium apps (aka. application subscriptions) (#6452)

* improve app subscription docs

* mention where premium required can't be used

Co-Authored-By: Donovan Daniels <hewwo@yiff.rocks>

* Update docs/monetization/SKUs.md

Co-authored-by: Colin Loretz <colin.loretz@discordapp.com>

* fix table

* restore consumed (for now)

---------

Co-authored-by: Donovan Daniels <hewwo@yiff.rocks>
Co-authored-by: Colin Loretz <colin.loretz@discordapp.com>

seailz committed:
fixed merge conflicts (#5669)

pedrofracassi committed:
Update OAuth2.md to clarify application.commands scope (#6324)

* Update OAuth2.md

* Update docs/topics/OAuth2.md

---------

Co-authored-by: shay <swdewael@gmail.com>

Th3-M4jor committed:
Fix incorrect documentation of starting a thread in a forum channel (#5478)

* correct position of files[n]/payload_json

* files[n] is optional

* fix link

---------

Co-authored-by: Colin Loretz <colin.loretz@discordapp.com>

Jupith committed:
Add some fired events on widget endpoints (#5767)

* Add some gateway events fired from rest actions

* Add specificity for Invite Create event: GET Guild Widget

---------

Co-authored-by: Colin Loretz <colin.loretz@discordapp.com>

colinloretz committed:
Followup #5767, when INVITE_CREATE is emitted (#6472)

stevenpetryk committed:
Add @mdx-js/react as a peer dependency (#6480)

colinloretz committed:
Premium Apps: SKU and Entitlement fields (#6477)

* Remove consumed from entitlements as not usable by devs

* Reorder sku fields to match sku example json

* Document 'deleted' field on SKUs

* Update 'before?' and 'after?' desc for entitlement pagination

* Add to SKU flags

* Add ENTITLEMENT_* to Gateway Events

* Update ENTITLEMENT_* gateway event descriptions

* Clarify ENTITLEMENT_UPDATE and recurring subscriptions

* Fix inconsistency in PREMIUM_REQUIRED docs

colinloretz committed:
Added changelog for RL discordapp.com (#6491)

Lulalaby committed:
fix changelog entry (#6492)

colinloretz committed:
Change Log: Premium Apps now available in EU and UK (#6494)

* Update docs for UK/EU

* Formatting RL changelog fix

* EU/UK premium apps changelog

Lulalaby committed:
fix: monetization docs (#6499)

* fix: monetization overview - remove test app

* add missing error code 40074

* add missing error code 50057

* fix: inconsistency for values / types

Commit:
edited: Fixed Markdown scrolling (#6486)

2. Instead of using the term "2.", it was written as "second".
https://i.imgur.com/pYrOBAE.png

Commit:
Fixed: Warning and Information boxes (#6487)

According to the current article, warning/information boxes were not visible. But with this update they are!

EN: https://i.imgur.com/92uLV90.png 
TR: https://i.imgur.com/WJmE0CW.png

jaw0r3k committed:
fix: do not translate blocks (#6488)

shaydewael committed:
message edit permissions change log (#6513)

* interactions followup permissions changelog

* fix link

* interaction response

hammerdr committed:
[onboarding] Clarify how emoji are currently accepted in update and create endpoints (#6479)

* [onboarding] Clarify how emoji are currently accepted in update and
create endpoints

There's some drift here from some later changes from when we first
created the API. I've added a note for us to go clean this up at
some point, but because it isn't high priority let's change the docs
to match what it happening today.

* Update docs/resources/Guild.md

* Update docs/resources/Guild.md

* Update docs/resources/Guild.md

* Update docs/resources/Guild.md

* Update docs/resources/Guild.md

* Update docs/resources/Guild.md

---------

Co-authored-by: shay <swdewael@gmail.com>

shaydewael committed:
Remove dispatch documentation (#6530)

* remove dispatch docs

* remove other dispatchy things

advaith1 committed:
Document permission splits for expressions and events (#6305)

* Document permission splits for expressions and events

* Added info callout for docs delay

---------

Co-authored-by: Colin Loretz <colin.loretz@discordapp.com>

dependabot[bot] committed:
Bump actions/setup-node from 3 to 4 (#6498)

Bumps [actions/setup-node](https://github.com/actions/setup-node) from 3 to 4.
- [Release notes](https://github.com/actions/setup-node/releases)
- [Commits](https://github.com/actions/setup-node/compare/v3...v4)

---
updated-dependencies:
- dependency-name: actions/setup-node
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

mccoderpy committed:
fix: Make description of `exclude_ended` field for list entitlements endpoint more specific (#6516)

(actually describing wich entitlements will be omitted)

colinloretz committed:
Fix thread/forum channel docs regression (#6546)

* Fix thread content regression

* Update docs/resources/Channel.md

Co-authored-by: Luca Kellermann <47486203+lukellmann@users.noreply.github.com>

* tables and rerun actions

---------

Co-authored-by: Luca Kellermann <47486203+lukellmann@users.noreply.github.com>

colinloretz committed:
Premium App Subscriptions: Testing guidance (#6548)

* Update to testing docs

* Changelog for testing entitlements

* bump date

* Clarify test entitlement deletion

* fix description for exclude_ended?

* Update docs/Change_Log.md

Co-authored-by: Jiralite <33201955+Jiralite@users.noreply.github.com>

---------

Co-authored-by: Jiralite <33201955+Jiralite@users.noreply.github.com>

colinloretz committed:
Changelog: Experimenting w/ E2EE Voice & Video (#6554)

* e2ee experiment changelog

* Update docs/Change_Log.md

Co-authored-by: Zoddo <github@zoddo.fr>

---------

Co-authored-by: Zoddo <github@zoddo.fr>

advaith1 committed:
Document applied_tags on Execute Webhook (#6560)

colinloretz committed:
Changelog: Clarification on Permission Splits for Expressions and Events (#6578)

* Rollback previous perm split changelog

* Add dev note to CREATE_GUILD_EXPRESSIONS and CREATE_EVENTS

shaydewael committed:
add fields limit (#6585)

KernelFreeze committed:
Added `Spanish, LATAM` to `Reference.md` (#6577)

* Added `Spanish, LATAM` to `Reference.md`

* Updated Language Name too

shaydewael committed:
Add Attachment CDN params (#6650)

* draft attachment CDN change

* ye

ypisetsky committed:
[message_create] Document new `enforce_nonce` param (#6647)

Recently I added a new enforce_nonce on POST channel/messages parameter that will allow bots to opt in to having their messages deduped on the nonce parameter in a short duration.

dependabot[bot] committed:
Bump @typescript-eslint/parser from 6.7.0 to 7.0.1 (#6655)

* Bump @typescript-eslint/parser from 6.7.0 to 7.0.1

Bumps [@typescript-eslint/parser](https://github.com/typescript-eslint/typescript-eslint/tree/HEAD/packages/parser) from 6.7.0 to 7.0.1.
- [Release notes](https://github.com/typescript-eslint/typescript-eslint/releases)
- [Changelog](https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/parser/CHANGELOG.md)
- [Commits](https://github.com/typescript-eslint/typescript-eslint/commits/v7.0.1/packages/parser)

---
updated-dependencies:
- dependency-name: "@typescript-eslint/parser"
  dependency-type: direct:development
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

* bump related deps

---------

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>
Co-authored-by: Maisy Linning <maisy.linning@discordapp.com>

AlmostSuspense committed:
Update character limit for slash commands (#6668)

* Update Application_Commands.md

* Fix command limit

---------

Co-authored-by: Dominik K <dominik@koch-bautechnik.de>

bremea committed:
GameSDK Relationships now requires OAuth (#6671)

codeofandrin committed:
Add `SUPPRESS_NOTIFICATIONS` to interaction callback flags (#5976)

* Add `SUPPRESS_NOTIFICATIONS` to interaction callback flags

* Fix table formatting

* Add `SUPPRESS_NOTIFICATIONS` to execute webhook flags

* Fix table formatting

* Add comma to listing

* Removed comma

* Reformat tables

---------

Co-authored-by: Jeff Cailteux <jkcailteux@users.noreply.github.com>

ayubun committed:
elevate required permissions for guild prune (#6685)

shaydewael committed:
Revert "elevate required permissions for guild prune (#6685)" (#6687)

This reverts commit 8de8a02f4413053d79c7617373126e7285a6d693.

amatsagu committed:
fix: update uri for Tempest lib in community resources (#6626)

I've recently renamed all my projects to use lowercase titles so it's more aligned with typical github project. Old redirection will only work for week or two so it shou;d be updated till then. Thanks!

advaith1 committed:
Document GitHub webhook supported events (#6586)

colinloretz committed:
Developer Preview Docs for Activities (#6722)

* Activities + Embedded App SDK docs

* Added changelog for Activities

shaydewael committed:
Add docs for user-installed apps (#6723)

add user install docs + new pages for interactions and overview of apps

colinloretz committed:
Rearrange/backlinks to Game SDK docs  (#6724)

* Rename Game SDK starter page

* Added note pointing to Activity docs. Will revisit Game SDK docs soon!

* Update links to SDK Starter Guide

shaydewael committed:
remove TODOs (#6725)

shaydewael committed:
fix optionality for global commands (#6726)

FedorLap2006 committed:
Add integration_types_config to params of Application Edit endpoint (#6727)

ayubun committed:
add `MANAGE_GUILD` requirement for guild prune (#6688)

Co-authored-by: shay <swdewael@gmail.com>

shaydewael committed:
clarify to command create (#6728)

colinloretz committed:
Update Monetization Policy

colinloretz committed:
monetization policy update take 2

turkwr committed:
Invalid link fixed (#6747)

Lulalaby committed:
fix: embedded app sdk code example commands.commands typo (#6736)

Commit:
Fix OAuth2 Scope `guilds.members.read` (#6741)

alula committed:
Fix typo in example code (#6731)

Commit:
Fix `GetActivityInstanceConnectedParticipantsResponse` type (#6742)

ndzn committed:
Fix Full URL link (#6732)

ckohen committed:
ci: check mdx components (#6735)

infinitestory committed:
Update starter activity code to remove xss (#6764)

kpodp0ra committed:
Document `integration_type` oauth parameter (#6738)

* Document `integration_type` oauth parameter

* Update OAuth2.md

---------

Co-authored-by: Yoni Lerner <yonilerner@gmail.com>

MinnDevelopment committed:
Document bulk ban endpoint (#6720)

* Document bulk ban endpoint

* Add manage guild permission requirement

Co-authored-by: advaith <advaithj1@gmail.com>

* Update docs/resources/Guild.md

Co-authored-by: Adrian Paschkowski <adrian.paschkowski@ruhr-uni-bochum.de>

* Fix nullability

---------

Co-authored-by: advaith <advaithj1@gmail.com>
Co-authored-by: Adrian Paschkowski <adrian.paschkowski@ruhr-uni-bochum.de>

advaith1 committed:
Fix link to Uploading Files (#6768)

yonilerner committed:
App command integration_types/contexts nullability (#6765)

* App command integration_types/contexts nullability

* fix: wrong table row (#6766)

* fix: wrong table row

* yet another fix

---------

Co-authored-by: Lala Sabathil <aiko@aitsys.dev>

Commit:
Add bungie connection (#6716)

shaydewael committed:
User install edits (#6782)

* oauth2_install_params optional

* wording

* add preview language

* tables

Lulalaby committed:
fix: merge conflicts (#6783)

colinloretz committed:
Added CSV Export for Premium App Analytics (#6784)

tpcstld committed:
Document polls. (#6746)

matthova committed:
Add note and reference known issue of requests to external ports being remapped (#6814)

* Add note and reference known issue of requests to external ports being remapped

* Split port considerations into two sections

* More specific reference to iframes for requests instead of "activities"

advaith1 committed:
fix patchUrlMappings code example (#6808)

Commit:
Fix invalid MDX tag

JustinBeckwith committed:
build: validate mdx and md (#6816)

azycheng committed:
Replace user_id with user object on message interaction metadata (#6815)

hemu committed:
Update permissions needed for PATCH member.flags field (#6817)

hemu committed:
Add closing table column marker for Guilds table (#6819)

Add closing column marker and reformat table.

colinloretz committed:
Premium Apps: One-Time Purchase SKUs + Managing Your Store (#6820)

* Added docs for One-Time Purchase + Store

* Update docs/monetization/One-Time_Purchases.md

Co-authored-by: shay <swdewael@gmail.com>

* Update docs/Change_Log.md

Co-authored-by: shay <swdewael@gmail.com>

* Update docs/monetization/Managing_Your_Store.md

Co-authored-by: shay <swdewael@gmail.com>

* Update docs/monetization/One-Time_Purchases.md

Co-authored-by: shay <swdewael@gmail.com>

---------

Co-authored-by: shay <swdewael@gmail.com>

JustinBeckwith committed:
Wrap message formats in code blocks (#6827)

Commit:
feat: remove autocode tools, domain for sale (#6829)

advaith1 committed:
fix defaultAvatarIndex calculation in activity code examples (#6809)

lytefast committed:
[docs] Message Forwards feature (#6818)

Add documentation for message forwards.

A message forward gives users the ability to send a snapshot of a message in one channel to another channel.
- snapshots are immutable, and do not receive updates from the original message (unlike replies)
- similar to a REPLY message, a forward is created using a `message_reference`

[DDevs #api-announcement](https://canary.discord.com/channels/613425648685547541/697138785317814292/1233463756160503859) - thread there

# Examples

## Creates
```json
{
    message_reference": {
        "type": 1,  // FORWARD
        "message_id": "123456789012345678",
        "channel_id": "123456789012345678",
        "guild_id": "123456789012345678"
    },
}
```

## Responses
```json
{
    "author": {
        "id": 123456789012345678,
    },
    "message_reference": {
        "type": 1,  // FORWARD
        "message_id": "123456789012345678",
        "channel_id": "123456789012345678",
        "guild_id": "123456789012345678"
    },
    "message_snapshots": [
        {
            "content": "hello world",
            "embeds": [
                // Embed objects
            ],
            "attachments": [
                // Attachment objects
            ]
        }
    ]
}
```

shaydewael committed:
Revert "[docs] Message Forwards feature (#6818)" (#6832)

This reverts commit 57a5077009d8e7c0a2ac1937aba63a6a86d16028.

JustinBeckwith committed:
Add callout for sticker gif urls (#6831)

advaith1 committed:
Update markdown table CI to Node 20 (#6845)

Wertzui123 committed:
Fix a typo (wrong footnote index) (#6838)

Nonolanlan1007 committed:
Fixed code example for setConfig() command (#6841)

Zoom-Developer committed:
Add DiscordSDK.close() to Embedded App Sdk (#6770)

MazeXP committed:
Link to actuall poll object in message structure (#6828)

IllagerCaptain committed:
Add error code 40333 (#6557)

Kimii55 committed:
Getting Started has been corrected a bit. (#6584)

Some text has been changed, but most importantly everything that is a (button, buttons) has been changed to (Schaltfläche, Schaltfächen) as these words are used to name a button.

Doc94 committed:
Add missing me_burst in message example (#6489)

Lulalaby committed:
docs: document connection type domain (#6395)

* added connection type domain

* Fix order

Co-authored-by: advaith <advaithj1@gmail.com>

---------

Co-authored-by: advaith <advaithj1@gmail.com>

nathanctech committed:
Read-only members can invite bots (#6408)

Read-only (and by extension, admin/owner) can invite bots marked private. Clarifying this as it may not be apparent.

Renegade334 committed:
Add X-Audit-Log-Reason to Follow Announcement Channel endpoint (#6559)

advaith1 committed:
Link boolean query param types to reference section (#6844)

advaith1 committed:
Update avatar decoration docs (#6464)

* Update avatar decoration docs

* add to example user

* oh is this the standard formatting now

* add guild member avatar decoration field

* Update docs/resources/User.md

---------

Co-authored-by: shay <swdewael@gmail.com>

wasdennnoch committed:
Remove outdated notice of interaction @everyone mention permissions (#5501)

* Remove outdated notice of interaction @everyone permissions

* Update docs/interactions/Receiving_and_Responding.md

Co-authored-by: shay <swdewael@gmail.com>

---------

Co-authored-by: shay <swdewael@gmail.com>

nathanctech committed:
Update upgrading to slash commands tutorial (#6360)

* Update Upgrading_to_Application_Commands.md

Update wording since `application.commands` is included in the `bot` scope automatically.

* Update docs/tutorials/Upgrading_to_Application_Commands.md

Co-authored-by: splatter <splatterxl@outlook.ie>

* Update docs/tutorials/Upgrading_to_Application_Commands.md

---------

Co-authored-by: splatter <splatterxl@outlook.ie>
Co-authored-by: shay <swdewael@gmail.com>

shaydewael committed:
update integration update
